front/app/containers/Admin/projects/project/nativeSurvey/FormResults/Files.tsx

```diff
@@ -50,7 +50,13 @@ const Files = ({ files }: FilesProps) => {
               start={virtualItem.start}
             >
               <Text m="0px" px="24px">
-                <a href={files[virtualItem.index].url} target="_blank">{files[virtualItem.index].name}</a>
+                <a
+                  href={files[virtualItem.index].url}
+                  target="_blank"
+                  rel="noreferrer"
```

Comment:
@jamesspeake Had this complaint: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md.

You might want to look into it as well.